tensorflow/core/platform/windows/port.cc

```diff
@@ -149,11 +149,16 @@ bool Snappy_Uncompress(const char* input, size_t length, char* output) {
 string Demangle(const char* mangled) { return mangled; }
 
 double NominalCPUFrequency() {
-#ifdef TENSORFLOW_USE_ABSL
-  return absl::base_internal::NominalCPUFrequency();
-#else
+  DWORD data;
+  DWORD data_size = sizeof(data);
+  #pragma comment(lib, "shlwapi.lib")  // For SHGetValue().
```

Comment:
If you want to avoid this `pragma`, then we will have to add `shlwapi.lib` to linker flags. For Bazel it will be `-defaultlib:shlwapi.lib` or Bazel will think `shlwapi.lib` is a label.

Reply:
I understand, but don't we need `#include <shlwapi.h>` as well to use `SHGetValue`

Reply:
Sorry for missing your point. Yes, `shlwapi.h` needs to be included explicitly, I wrongly assumed that it is already included in `windows.h`. Done.

Reply:
Thank you!